docs/changelog.md

```diff
@@ -24,9 +24,57 @@ use the latest versions of the software.
 | 0.3.z | 2 | 1.2.z, 2.0.z, 2.1.z, 2.2.z, 3.0.z, 3.11.z | 1.0.z, 1.1.z, 1.2.z, 1.3.z, 1.4.z | 1.0.0, 1.1.0, 1.1.2, 1.2.0, 1.3.0, 1.4.0 | 1.5-1.7.z, 2.3-2.4.z, 5.y, 6.y |  5.2-5.5.z, 6.2-6.6.z, 7.y | 3.3.z | 2.2.z | 2.11.z |
 | 0.4.z | 2 | 2.1.z, 2.2.z, 3.0.z, 3.11.z | 1.2.z, 1.3.z, 1.4.z, 2.1.z | N/A | 5.y, 6.y | 7.y | 3.4.z | 2.2.z | 2.11.z |
 | 0.5.z | 2 | 2.1.z, 2.2.z, 3.0.z, 3.11.z | 1.2.z, 1.3.z, 1.4.z, 2.1.z | 1.3.0, 1.4.0, 1.5.z, 1.6.z, 1.7.z, 1.8.z, 1.9.z, 1.10.z, 1.11.z, 1.14.z | 6.y, 7.y | 7.y | 3.4.z | 2.2.z | 2.11.z | 
+| 0.6.z | 2 | 2.1.z, 2.2.z, 3.0.z, 3.11.z | 1.2.z, 1.3.z, 1.4.z, 2.1.z | 1.3.0, 1.4.0, 1.5.z, 1.6.z, 1.7.z, 1.8.z, 1.9.z, 1.10.z, 1.11.z, 1.14.z | 6.y, 7.y | 7.y | 3.4.z | 2.2.z | 2.11.z | 
 
 ## Release Notes
 
+### Version 0.6.0 (Release Date: X, 2020)
+
+```xml tab='Maven'
+<dependency>
+    <groupId>org.janusgraph</groupId>
+    <artifactId>janusgraph-core</artifactId>
+    <version>0.6.0</version>
+</dependency>
+```
+
+```groovy tab='Gradle'
+compile "org.janusgraph:janusgraph-core:0.6.0"
+```
+
+**Tested Compatibility:**
+
+* Apache Cassandra 2.2.10, 3.0.14, 3.11.0
+* Apache HBase 1.2.6, 1.3.1, 1.4.10, 2.1.5
+* Google Bigtable 1.3.0, 1.4.0, 1.5.0, 1.6.0, 1.7.0, 1.8.0, 1.9.0, 1.10.0, 1.11.0, 1.14.0
+* Oracle BerkeleyJE 7.5.11
+* Elasticsearch 6.0.1, 6.6.0, 7.6.1
+* Apache Lucene 7.0.0
+* Apache Solr 7.0.0
+* Apache TinkerPop 3.4.6
+* Java 1.8
+
+For more information on features and bug fixes in 0.6.0, see the GitHub milestone:
+
+-   <https://github.com/JanusGraph/janusgraph/milestone/13?closed=1>
+
+#### Upgrade Instructions
+    
+##### Removal of Cassandra Thrift support
+Thrift will be completely removed in the Cassandra 4.
+All deprecated Cassandra Thrift backends were removed in JanusGraph 0.6.0 
+to allow JanusGraph contributors focus more on new features and 
+major improvements instead of maintaining deprecated versions of old backends.
+Therefore, JanusGraph contributors added basically all the functionality 
+to Cassandra CQL backend for easy migration, 
+see [migration guide](advanced-topics/migrating-thrift.md).
+Removed backends: `cassandrathrift`, `cassandra`, `astyanax`, `embeddedcassandra`
+
+!!! note 
+    Special Repo for all decrepated Cassandra backends
+    We will create a [repo](https://github.com/JanusGraph/janusgraph-cassandra) for Cassandra Thrift and Cassandra Embedded, 
```

Comment:
Do we really want to encourage people to provide contributions to this repo? I'd see it as basically read-only and only accept PRs for bugs that really need to be fixed.

How about something like this instead?

>The source code for the Thrift backends will be moved into a [dedicated repository](https://github.com/JanusGraph/janusgraph-cassandra).
While we do not support them any more, users can still use them if they for some reason cannot migrate to CQL.

We of course need to update this again to remove the _will_ here after moving the code there.